guide/13-managing-arcgis-applications/hub-basic-guide.ipynb

```diff
@@ -0,0 +1,744 @@
+{
```

Comment:
<p>Since this is a guide for Hub Basic, I think the <em>If you're using ArcGIS Hub Premium, you can launch a site by creating an initiative </em>sentence is confusing. This guide is meant to demonstrate Hub Basic workflows, and it links to the Enterprise Sites guide above. I found this sentence unnecessary.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>